src/chttpd/src/chttpd_db.erl

```diff
@@ -1960,6 +1962,13 @@ validate_security_can_be_edited(DbName) ->
         {_,_} -> ok
     end.
 
+validate_revs(Doc, NewEdits) ->
+    if NewEdits =:= false andalso Doc#doc.revs =:= {0, []} ->
+        throw({bad_request, ?l2b("When `new_edits: false`, "
+        ++ "the document needs `_rev` or `_revisions` information")});
```

Comment:
A minor nit: It might sound better to say:
```
it needs `_rev` or `_revisions` specified
```

Reply:
Changed